src/webgpu/util/texture/texel_data.ts

```diff
@@ -505,6 +506,66 @@ export const kTexelRepresentationInfo: {
       },
       pack: components => packComponents([TexelComponent.Stencil], components, 8, 'uint'),
     },
+    'depth24unorm-stencil8': {
+      componentOrder: [TexelComponent.Depth, TexelComponent.Stencil],
+      componentInfo: {
+        Depth: {
+          dataType: 'unorm',
+          bitLength: 24,
+        },
+        Stencil: {
+          dataType: 'uint',
+          bitLength: 8,
+        },
+      },
+      encode: components => {
+        assert(components.Stencil !== undefined);
+        assertInIntegerRange(components.Stencil, 8, false);
+        return {
+          Depth: floatAsNormalizedInteger(components.Depth ?? unreachable(), 24, false),
+          Stencil: components.Stencil,
+        };
+      },
+      decode: components => {
+        assert(components.Stencil !== undefined);
+        assertInIntegerRange(components.Stencil, 8, false);
+        return {
+          Depth: normalizedIntegerAsFloat(components.Depth ?? unreachable(), 24, false),
+          Stencil: components.Stencil,
+        };
+      },
+      pack: components =>
```

Comment:
see below, probably can just be unreachable

Reply:
Done